app/controllers/api/v1/send_sms_controller.rb

```diff
@@ -102,45 +140,17 @@ def guardian_phone
     relationships = service.find_relationships(filters)
     relationships[0].relation.try(:person_attributes)
                     .try(:find_by, person_attribute_type_id: PersonAttributeType.find_by_name('Cell Phone Number').id)
-                    .try(:value)
+                    .try(:value).gsub(/\s+/, '')
   end
 
   def service
     PersonRelationshipService.new(Person.find(params[:person_id]))
```

Comment:
Let us require the person_id param because sometimes the frontend it not sending it

Reply:
I have made the changes to the function